### ISSUE TITLE
Auto complete should work in VS Code for exported endpoint and data classes

### ISSUE THREAD

**Issue by Artur-** [vaadin/flow#10192]
### Description of the bug / feature
If you create an endpoint class like
```java
@Endpoint
@AnonymousAllowed
public class ProductEndpoint {
    public List<Product> list() {
        return new ArrayList<>();
    }
}
```

you should be able to write a TS file like
```ts
const items = await Product<cmd-space>
```

I would expect this to show `ProductEndpoint` as an auto complete option, in addition to `Product` and `ProductModel`.

### Actual behavior
It shows `ProductModel` and nothing else

### Versions:
    - Vaadin / Flow version: 19 rc1

**Comment by Artur-** [comment]
It seems like this would be fixed by #7176  but as that is only available in 20.0-SNAPSHOT that does not work at the moment, I cannot say for sure

**Comment by Artur-** [comment]
Yes, seems to work now. Thanks
